src/pages/core/external-plugins/app-data.md

```diff
@@ -0,0 +1,248 @@
+---
+title: AppData Plugin
+metaTitle: Core - AppData Plugin
+description: Learn about the MPL Core AppData Plugin
+---
+
+## What is an AppData Plugin?
+
+The `AppData` external plugin stores and contains arbitrary data that can be written to by the `dataAuthority`. Note this is different then the overall plugin authority stored in the `ExternalRegistryRecord` as it cannot update/revoke authority or change other metadata for the plugin.
+
+Think of AppData as like a partition data area of an Asset that only a certain authority can change and write too.
+
+This is useful for 3rd party sites/apps to store data they made need to execute certain functionality within their product/app.
+
+## Works With
+
+|                       |     |
+| --------------------- | --- |
+| MPL Core Asset        | ✅  |
+| MPL Core Collection\* | ✅  |
+
+\* MPL Core Collections can also work with the LinkedAppData Plugin.
+
+## What is a LinkedAppData Plugin?
+
+The `LinkedAppData` plugin is built for Collections. It allows you to add a single plugin adapter on the collection which will allow you to write to any Asset in the collection.
+
+## Arguments
+
+| Arg           | Value                       |
+| ------------- | --------------------------- |
+| dataAuthority | PluginAuthority             |
+| schema        | ExternalPluginAdapterSchema |
+
+### dataAuthority
+
+{% dialect-switcher title="AttributeList" %}
+{% dialect title="JavaScript" id="js" %}
+
+```ts
+const dataAuthority = {
+  type: 'Address',
+  address: publicKey('11111111111111111111111111111111'),
+}
+```
+
+{% /dialect %}
+
+{% dialect title="Rust" id="rust" %}
+
+```rust
+use mpl_core::types::{PluginAuthority}
+
+let data_authority = Some(PluginAuthority::Address {address: authority.key()}),
+```
+
+{% /dialect %}
+{% /dialect-switcher %}
+
+### schema
+
+The schema determines the type of data that is being stored within the AppData plugin. Certain schemas will be DAS compatible and indexed by DAS RPC providers. These schema data types include:
```

Comment:
All data will be indexed by DAS, not just certain schemans

If it is binary data it will be saved as base64
If it is JSON it will be saved as JSON
If it is msgPack it will be saved as JSON

If there's an error trying to read it as JSON or MsgPack it will be saved as binary